compiler/src/main/scala/org/scalajs/core/compiler/PrepJSExports.scala

```diff
@@ -726,7 +552,6 @@ trait PrepJSExports { this: PrepJSInterop =>
   /** Whether a symbol is an annotation that goes directly on a member */
   private lazy val isDirectMemberAnnot = Set[Symbol](
```

Comment:
Looks like this should be renamed to `isExportAnnot` in the first commit, shouldn't it?